package.json

```diff
@@ -54,7 +54,7 @@
     "@radix-ui/react-select": "^2.2.4",
     "@radix-ui/react-slot": "^1.2.2",
     "@radix-ui/react-switch": "^1.2.4",
-    "@radix-ui/react-toast": "^1.2.13",
+    "@radix-ui/react-toast": "^1.2.14",
```

Comment:
why would we need this 14 change? doesn't the `^` pick minor / patches? Asking this as we might want to update other radix ui packages as well? 🤔